_data/simple-icons.json

```diff
@@ -9106,6 +9106,11 @@
             "hex": "4ACBD6",
             "source": "https://www.tp-link.com/"
         },
+        {
+            "title": "tqdm",
+            "hex": "FFC107",
+            "source": "https://img.tqdm.ml/logo.svg"
```

Comment:
```suggestion
            "source": "https://github.com/tqdm/img/blob/0dd23d9336af67976f88f9988ea660cde78c54d4/logo.svg"
```

Reply:
doesn't seem to be consistently used by other GH-hosted logos but whatever...

Reply:
To clarify as to why I made that suggestion: with no website at http://tqdm.ml/, I'd be worried about the possibility of link rot in the future, making it harder for other contributors to find a suitable source when reviewing this one down the road.